### DISCUSSION TITLE
How to put it in my system?

### DISCUSSION THREAD

**Discussion by baibhavKumar1** [Nikorasu/LiveWhisper#3] [Q&A]
What's the process of installing it and what are the processes involved before you can say "assistant's name"?

**Comment by Nikorasu** [comment]
You should just be able to install Whisper, same as their instructions on their github, then drop my files into the same directory you have the virtual environment setup in. I haven't updated my Whisper files in a while, so I'm not sure if something has changed recently..
If you need more instruction, let me know.
